README.md

```diff
@@ -2732,7 +2732,7 @@ using data on professional baseball players.  (Dataset courtesy of the
 UCLA Statistics Dept.)
 
 ``` r
-> mlb <- read.table('https://raw.githubusercontent.com/matloff/fasteR/master/data/mlb.txt',header=TRUE)
+> mlb <- read.table('https://raw.githubusercontent.com/matloff/fasteR/master/data/mlb.txt',header=TRUE,stringsAsFactors=TRUE)
```

Comment:
Newer versions of R doesn't do this automatically, while the rest of the code assumes some of the variables are factors.